setup.py

```diff
@@ -0,0 +1,50 @@
+import os
+import imp
+
+from setuptools import setup, find_packages
+
+with open("README.md") as f:
+    readme = f.read()
+
+
+version_file = os.path.abspath("pyblish_standalone/version.py")
+version_mod = imp.load_source("version", version_file)
+version = version_mod.version
+
+
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 2",
+    "Programming Language :: Python :: 2.6",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3",
```

Comment:
I wouldn't include compatibility with Python 3 unless you are also testing with Python 3 which I suspect you are not (the hint is in the print-statements).

Problem is, this will show up for people trying to pip install this with Python 3 and they will get furious and confused, rip the liquid crystals out of their screens and shout breathlessly into the night. And I just don't want them the hurt themselves.